common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -22,18 +22,10 @@
 import java.security.PrivateKey;
 import java.security.Provider;
 import java.util.Properties;
-import javax.net.ssl.HttpsURLConnection;
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLContextSpi;
-import javax.net.ssl.SSLEngine;
-import javax.net.ssl.SSLEngineResult;
-import javax.net.ssl.SSLException;
-import javax.net.ssl.SSLServerSocketFactory;
-import javax.net.ssl.SSLSessionContext;
-import javax.net.ssl.SSLSocket;
-import javax.net.ssl.SSLSocketFactory;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.X509TrustManager;
+//import javax.net.ssl.HttpsURLConnection.HostnameVerifier; THIS IS GIVING ISSUES FOR SOME REASON
+import javax.net.ssl.*;
```

Comment:
No wildcard imports, please.  I suspect this was your IDE being "helpful"... For IntelliJ, this [post](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-) points to the relevant setting.

Reply:
Thanks for catching this!